scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -180,6 +180,8 @@ elif [ "$os" == "ubuntu20.04" ] ; then
         gstreamer1.0-plugins-ugly
         gstreamer1.0-vaapi
         gstreamer1.0-tools
+        gstreamer1.0-x
+        intel-media-va-driver-non-free
```

Comment:
> intel-media-va-driver-non-free

Please check for conflicts of this package with Media SDK component from OpenVINO packages.

/cc @adranit

Reply:
I removed this change with driver installation.
We will do it as a separate PR.

Reply:
We will add it to data_runtime image it doesn't contain media SDK

Reply:
We have agreed that the driver we need for runtime will be installed by default in docker images.